common/app/views/fragments/contentMeta.scala.html

```diff
@@ -56,7 +56,7 @@
     @if(item.tags.contributors.length == 1 && item.tags.contributors.headOption.exists(_.properties.twitterHandle.nonEmpty)) { content__meta-container--twitter}
     @if(item.tags.contributors.length == 1 && item.tags.contributors.headOption.exists(_.properties.emailAddress.nonEmpty)) { content__meta-container--email}">
 
-    @if(showBadge) {
+    @if(showBadge && (NewCommercialContent.isSwitchedOff || (NewCommercialContent.isSwitchedOn && (item.tags.isVideo || item.tags.isAudio || (item.tags.isArticle && !item.commercial.isAdvertisementFeature))))) {
```

Comment:
Logically, condition is same as `showBadge && (NewCommercialContent.isSwitchedOff || (item.tags.isVideo || item.tags.isAudio || (item.tags.isArticle && !item.commercial.isAdvertisementFeature)))`

But it's not clear what this condition means.  If the switch is off, are we only going to show logos on video pages, audio pages and non-paid content?